src/amo/components/SearchResults.js

```diff
@@ -29,10 +30,20 @@ class SearchResults extends React.Component {
     } = this.props;
     const { query } = filters;
 
+    let loadingMessage;
     let messageText;
 
     if (loading) {
-      messageText = i18n.gettext('Searching...');
+      loadingMessage = (
+        <div className="LoadingIndicator-container"
+          ref={(ref) => { this.LoadingIndicatorContainer = ref; }}>
+          <LoadingIndicator>
+            <span className="visually-hidden">
+              {i18n.gettext('Searching…')}
```

Comment:
Yeah, it's okay. The triple dots were there before and that's worse (and we use ellipsis elsewhere).